ruby/game/game_spec.rb

```diff
@@ -5,26 +5,24 @@
 
   it "resets guess count on initialization, prompts user 1 to enter word for user 2 to guess" do
     new_game.word_to_guess
-    expect(new_game.word_to_guess).to eq @word
+    expect(new_game.word_to_guess).to eq ""
   end
 
   it "takes user input and checks guess to see if correct" do
-    new_game.check_input(guess)
-    expect(new_game.check_input(guess)).to eq @is_over
+    expect(new_game.check_input("guess")).to eq ["guess"]
   end
 
   it "gives user hint on letter guesses" do
-    new_game.hint(guess)
-    expect(new_game.hint(guess)).to eq "Your word includes '#{letter}'!"
+    new_game.hint("")
```

Comment:
you can check hint better by creating an instance of the word to be guessed.  then you can actually feed in letters and test the result.